libc/src/__support/integer_utils.h

```diff
@@ -15,6 +15,10 @@
 #include "bit.h"
 #include "number_pair.h"
 
+#ifndef __SIZEOF_INT128__
```

Comment:
this include should be unconditional since the function is always used.

Reply:
Done